### PR TITLE
[RTL-1980][RTL-1860][RTL-2424]

### PR DESCRIPTION
**Pull request by rshii**
- Reads and forwards Epic 880 printer error statuses
- Adds utility for checking ASB capability
- Formats barcodes for kiosk printed barcodes for easy scanning
- Stops syslog and kernel log spam due to using device without claiming interface
- Bumps version number

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -22,6 +22,8 @@ rusb = "0.9"
 thiserror = "1.0.40"
 qrcode =  { version = "0.12", optional = true }
 log = "0.4"
+rayon = "=1.7.0"
```
Comment (flynnguy) [inline Cargo.toml]: Are these being used? Or are these left over from something else you were trying?

Comment (rshii) [reply Cargo.toml]: Yeah I was trying something that pulled this dependency in with the latest not compatible with the version of rust we're using, so I pinned this one. Reverted.

**Review by flynnguy — APPROVED** [review]

File: Cargo.toml (hunk 1 above)
Comment (don) [reply Cargo.toml]: If you update https://github.com/flynnguy/posify/blob/master/rust-toolchain can you unpin it?

Comment (rshii) [reply Cargo.toml]: Yeah, I'll delete rust-toolchain, I don't think we need it anymore

**Pull request merged**